libs/build_nceplibs.sh

```diff
@@ -223,6 +246,17 @@ VERBOSE=$MAKE_VERBOSE make -j${NTHREADS:-4}
 [[ $USE_SUDO =~ [yYtT] ]] && sudo -- bash -c "export PATH=$PATH; make install" \
                           || make install
 
+cd ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+
+# Install CRTM fix files
+if [[ "$name" == "crtm" ]]; then
+  if [[ ${STACK_crtm_install_fix:-} =~ [yYtT] ]]; then
+    if [[ -d crtm_fix-$version ]]; then
+      ./link_crtm_coeffs.sh ./crtm_fix-$version $prefix/fix
```

Comment:
Does there need to be a check here if it's already installed?

Reply:
how will it be already installed? `$prefix` is the location of the library, which is protected by `OVERWRITE`.
This is the problem remember, the coefficients living with the library.

Reply:
Right, the top-level overwrite is there.